modules/administration-guide/partials/proc_setting-up-the-bitbucket-application-link.adoc

```diff
@@ -0,0 +1,100 @@
+:_content-type: PROCEDURE
+:description: Setting up the Bitbucket application link
+:keywords: configuring-authorization, configure-authorization, private-repository, private-git-repository, private-repo, private-git-repo, private-bitbucket, private, bitbucket, bitbucket-repo, bitbucket-repository
+:navtitle: Setting up the Bitbucket application link
+// :page-aliases:
+
+[id="setting-up-the-bitbucket-application-link_{context}"]
+= Setting up the Bitbucket application link
+
+Set up a Bitbucket application link as follows:
+
+.Prerequisites
+
+* You are logged in to Bitbucket.
+* link:https://www.openssl.org/[`openssl`] is installed in the operating system you are using.
+* link:https://www.gnu.org/software/coreutils/base64[`base64`] is installed in the operating system you are using.
+
+.Procedure
+
+. On a command line, generate the `privatepkcs8-stripped.pem` key in the required format and with Base64 encoding:
++
+[source,subs="+quotes,+attributes,+macros"]
+----
+$ openssl genrsa -out private.pem 2048
+
+$ openssl pkcs8 -topk8 -inform pem -outform pem -nocrypt -in private.pem -out privatepkcs8.pem
+
+$ cat privatepkcs8.pem | sed 's/-----BEGIN PRIVATE KEY-----//g' |  sed 's/-----END PRIVATE KEY-----//g' | tr -d '\n' | base64 > privatepkcs8-stripped.pem
```

Comment:
```suggestion
$ cat privatepkcs8.pem | sed 's/-----BEGIN PRIVATE KEY-----//g' |  sed 's/-----END PRIVATE KEY-----//g' | tr -d '\n' | base64 | tr -d '\n' > privatepkcs8-stripped.pem
```
Since the output of `base64` has newlines, I think it would be great to have an extra `tr -d '\n'` before writing to a file, since the user would have to remove newlines when pasting the content into the secret anyways.